src/rgw/rgw_sal_motr.cc

```diff
@@ -3926,10 +3924,19 @@ int MotrMultipartUpload::init(const DoutPrefixProvider *dpp, optional_yield y,
   rc = store->create_motr_idx_by_name(obj_part_iname);
   if (rc == -EEXIST)
     rc = 0;
-  if (rc < 0)
+  if (rc < 0) {
     // TODO: clean the bucket index entry
     ldpp_dout(dpp, 0) << "Failed to create object multipart index  " << obj_part_iname << dendl;
+    return rc;
+  }
 
+  // initialize with size=0 & object count=1 
+  rc = update_bucket_stats(dpp, store, owner.get_id().to_str(), tenant_bkt_name, 0, 0, 1, true);
+  if (rc != 0) {
+    ldpp_dout(dpp, 20) <<__func__<< ": Failed stats substraction for the "
```

Comment:
please update the comment msg. this is not stats subtraction.
Msg can be `Failed to update object count for the bucket/obj=**`

Reply:
done